modules/installation-proxy-configure-object.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-configure-object_{context}"]
+= Enabling cluster-wide proxy
+
+The proxy object is used to manage cluster-wide egress proxy. When a cluster is
+installed or upgraded without proxy configured, a proxy object is still
+generated but it will have a nil spec and status. For example:
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
+  creationTimestamp: "2019-08-21T22:36:49Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "24913"
+  selfLink: /apis/config.openshift.io/v1/proxies/cluster
+  uid: 2a344b01-c464-11e9-b3f3-025de4b59c38
+spec:
+  trustedCA:
+    name: ""
+status:
+----
+
+A cluster administrator can configure proxy for {product-title} by modifying
+this `cluster` proxy object.
+
+NOTE: Only the proxy object named `cluster` is supported, and no additional
+proxies can be created.
+
+.Prerequisites
```

Comment:
I've been using:

```
* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
* You must log in to the cluster as a cluster administrator.
```

Did we ever decide upon a common style for these frequently used prerequisites?

Reply:
I don't think so, I've just been consistent when I used them. And honestly I hate the "commonly known as `oc`" one. We did agree awhile back that we can call it the "OpenShift CLI (`oc`)", and I prefer that because it is descriptive and concise. I don't have a strong opinion on the other. This one actually came from Brandi since this PR builds on hers, but I had usually been saying something like "Access to the cluster as a user with the `cluster-admin` role."